src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private OperationType operation;
+    private String fruit;
+    private int quantity;
+
```

Comment:
Consider adding a no-argument constructor to `FruitTransaction` class. While it's not a requirement of the checklist, this is a common practice for POJOs and can increase compatibility with various Java frameworks and libraries.